lib/server.js

```diff
@@ -1,3 +1,4 @@
+const path = require('path');
```

Comment:
since the `relative` function is the only one being used on `path`, please change this to:

```
const { relative } = require('path');
```

and update the line using it below